skietbaan_fe/src/App.js

```diff
@@ -51,16 +53,16 @@ class App extends Component {
     render () {
     // const { user } = this.state
         return (
-            /** * <div className='App'>
-            <ul style={{ listStyle: 'none' }}>{
-            // Loop over the object keys and render each key
-            Object.keys(user).map(key => renderLine(user, key))
-            }
-            </ul>
-            */
 
+            <Provider store = {store}>
             <div className="App">
 
+                {/** <button onClick={this.simpleAction}>Test redux action</button>
+                <button onClick={this.handleClick}>Endpoint Test</button>
+        <pre>{JSON.stringify(this.props)}</pre> */}
+
+
+
                 {/* <button onClick={this.simpleAction}>Test redux action</button>
```

Comment:
remove this GC or add TODO with description

Reply:
Sorted. Kindly review